buddyboss/buddyboss-platform
# T1805: Issue: Video embed not working when a text is added in replies of forum discussions - #57224

### All Submissions:


### Changes proposed in this Pull Request:


Fixes #336.

### How to test the changes in this Pull Request:
1. https://screencast-o-matic.com/watch/cqlXiqUSJx
2. check with the single embed URL.
3. check with multiple embed URL - embed shows for the first URL only just like the facebook( https://prnt.sc/q95vmi )

### Other information:
